examples/step-22/doc/intro.dox

```diff
@@ -473,33 +473,32 @@ program to see that indeed the number of CG iterations does not change as we
 refine the mesh).
 
 So all we need in addition to what we already have is the mass matrix on the
-pressure variables. We could do that by building this matrix on the
-side in a separate data structure. However, it is worth remembering
-that although we build the system matrix
+pressure variables and we will store it in a separate object. However, it is
+worth remembering that we never actually do matrix-vector products or any other
+operations that considers all blocks with the entire system matrix
```

Comment:
I think this comment is somewhat obsolete now. It was meant to explain why we can store the mass matrix in the bottom right block: because we never actually do matrix-vector products with the full matrix. But now, with the correct zero in the bottom right, this comment sort of lacks context -- I would just remove it.